test/DynamoCoreTests/SerializationTests.cs

```diff
@@ -357,6 +377,9 @@ public void SerializationNonGuidIdsTest(string filePath)
         {
             modelsGuidToIdMap.Clear();
             DoWorkspaceOpenAndCompare(filePath, "json_nonGuidIds", ConvertCurrentWorkspaceToNonGuidJsonAndSave, CompareWorkspacesDifferentGuids, SaveWorkspaceComparisonDataWithNonGuidIds);
+
+            //UploadToS3() from Ram;
```

Comment:
you will want to upload both the guid and non guid tests and all data files associated with those graphs so this should be called from the other serializationTest method as well.

Reply:
hmm, you may want to add a breakpoint here and check if this method is called repeatedly... I dont think you want to make 2000 separate upload calls to s3, I think you just want to upload the two folders once.

Reply:
If Dynamo is going to upload to S3, then the configurations should be moved to the config files or environment variables. Uploading to S3 is "sync" ing a folder on the machine with S3 storage location. So, one sync will get all the files in S3.

Reply:
@ramramps @smangarole thats a good point... I dont think running these tests should upload to s3 - then what happens when a dev runs the test locally? It should not upload to s3. this should be some post test step or at least set with a flag or something.

Reply:
Please make uploading part seperate of Dynamo code. For internal tool, we probably can argue about it. But this will be wrapped in DynamoTest nuget package and spread.

Reply:
another thought here is upload should be from the  script (job) after a successful build.